docs/docs/ref/cut.md

```diff
@@ -13,7 +13,9 @@ FirstGoal, !; SecondGoal
 ```
 ## DESCRIPTION
 
-Discards all choicepoints made since the parent goal started execution, including the choicepoints, if any, created by calling the parent goal. In the following two cases, a cut in `Condition` will remove all choicepoints created by the `Condition`, any subgoals to the left of the `Condition`, and the choicepoint for the parent goal.
+
+Discards all choice points made since the parent goal started execution, including the choice points, if any, created by calling the parent goal. In the following three cases, a cut in `Condition` will remove all choice points created by the `Condition`, any subgoals to the left of the `Condition`, and the choice point for the parent goal.
```

Comment:
Minor: this change undid the choicepoint corrections in 186b6dabd782e613711f0cf3efb0b58fccc694e2